Modules/CustomerHistory/Frames.lua

```diff
@@ -58,11 +58,10 @@ function CraftSim.CUSTOMER_HISTORY.FRAMES:Init()
     createContent(self.frame)
 end
 
-function CraftSim.CUSTOMER_HISTORY.FRAMES:AddCustomer(customer)
-    self:ResetDropdown(customer)
-end
-
 local function GetFallbackCustomer()
+    if CraftSim.CUSTOMER_HISTORY.FRAMES.lastCustomer then
+        return CraftSim.CUSTOMER_HISTORY.FRAMES.lastCustomer
+    end
```

Comment:
I don't think that we still need the loop with this?

Reply:
I think we still need it, when we open the customer history but haven’t received any messages beforehand. Then we do need to get the first customer from the history map

Reply:
Unless we would rather just show an empty history view and let the user select a specific customer before showing any history

Reply:
We do actually need it indeed, but because there's no logic to load the history. If you remove that part then the history will always be empty on `/reload` or login.

Reply:
Keeping the last selected customer through reload and login isn't a bad idea. Let me think on it for a bit !